test/e2e/subscription_e2e_test.go

```diff
@@ -219,7 +218,7 @@ var _ = Describe("Subscription", func() {
 		stableChannel := "stable"
 		mainCSV := newCSV(mainPackageStable, testNamespace, "", semver.MustParse("0.1.0-1556661347"), []apiextensions.CustomResourceDefinition{crd}, nil, nil)
 		updatedCSV := newCSV(updatedPackageStable, testNamespace, "", semver.MustParse("0.1.0-1556661832"), []apiextensions.CustomResourceDefinition{crd}, nil, nil)
-		updatedCSV.SetAnnotations(map[string]string{resolver.SkipPackageAnnotationKey: ">=0.1.0-1556661347 <0.1.0-1556661832"})
+		updatedCSV.SetAnnotations(map[string]string{"olm.skipRange": ">=0.1.0-1556661347 <0.1.0-1556661832"})
```

Comment:
I know this is a test so it doesn't matter but this const `SkipPackageAnnotationKey` still exists. May as well keeping using it instead of hardcoding.

Reply:
I decided that the end-to-end test shouldn't depend on this constant from the implementation. If its value is changed, the test should fail and need to be updated since it represents a change in behavior.